src/components/flat-table/flat-table-row/flat-table-row.d.ts

```diff
@@ -1,16 +1,22 @@
-import * as React from 'react';
+import * as React from "react";
 
 export interface FlatTableRowProps {
   /** Array of FlatTableHeader or FlatTableCell. FlatTableRowHeader could also be passed. */
   children: React.ReactNode;
-  /** Function to handle click event. If provided the Component could be focused with tab key. */
-  onClick?: (ev: React.SyntheticEvent) => void;
+  /** Allows the row to be expanded, must be used with the `subRows` prop. */
+  expandable?: boolean;
+  /** Area to click to open sub rows when expandable. Default is `wholeRow` */
+  expandableArea?: "wholeRow" | "firstColumn";
   /** Allows developers to manually control highlighted state for the row. */
   highlighted?: boolean;
+  /** Function to handle click event. If provided the Component could be focused with tab key. */
```

Comment:
Is this description correct?

Reply:
ts description should match the js description. Changing it is not in the scope of this task.